components/org.wso2.carbon.identity.api.server.organization.user.invitation.management/org.wso2.carbon.identity.api.server.organization.user.invitation.management.v1/src/main/resources/organization-user-invitation.yaml

```diff
@@ -0,0 +1,483 @@
+openapi: 3.0.0
+info:
+  version: 1.0.0
+  title: 'Private CIAM Cloud - Parent User Invitation API Definition'
+  description: |-
+    This is the RESTful API for parent organization user invitation for child organizations in CIAM Cloud. This API 
+    allows administrators to invite users, manage the invitations and invitees are allowed to accept the invitation.
+  contact:
+    name: WSO2
+    url: 'https://wso2.com/ciam-suite/private-ciam-cloud/b2b-ciam'
+    email: iam-dev@wso2.org
+  license:
+    name: Apache 2.0
+    url: 'http://www.apache.org/licenses/LICENSE-2.0.html'
+servers:
+  - url: 'https://localhost:9443/o/{organization-domain}/api/server/v1'
+    variables:
+      organization-domain:
+        default: 10084a8d-113f-4211-a0d5-efe36b082211
+paths:
+  /guests/invite:
+    post:
+      tags:
+        - Parent Organization User Invitation
+      summary: Invite a parent organization user to a child organization
+      description: |
+        Initiates an invitation to a user in the parent organization to onboard to the child organization.
+        This will be initiated from the child organization.
+        
+        <b>Scope required:</b> <br/> 
+        - internal_guest_mgt_invite_add
+      operationId: invitationTriggerPost
+      requestBody:
+        $ref: '#/components/requestBodies/InvitationRequestPayload'
+      responses:
+        '200':
```

Comment:
Since an invitation resource is created with this operation, 201 would be the matching status code

Reply:
Done